pages/partials/submission.ejs

```diff
@@ -11,7 +11,16 @@
         </span>
         <span><%- include('submissionStatus', {submission: submission}); %></span>
       </div>
-      <small>Submitted at <%- submission.formatted_date %></small>
+      <span class="small">
+        <% if (authz_data.user.uid == submission.uid) { %>
+          <span class="pl-submission-attribution">You</span>
+          submitted 
+        <% } else { %>
+          Submitted by
+          <span class="pl-submission-attribution"><%= submission.uid %></span>
```

Comment:
Could we add the name as well, so it would say `Submitted by Matthew West (mwest@illinois.edu) at ...`? The name is in `users.name`.

Reply:
Done!